lib/data_update_scripts/20200503134234_remove_github_issues.rb

```diff
@@ -0,0 +1,7 @@
+module DataUpdateScripts
+  class RemoveGithubIssues
+    def run
+      GithubIssue.in_batches.delete_all
```

Comment:
this is to cleanup the status quo in the DB. 
We have a bunch of rows that have an invalid URL, which now that we fixed the URL will result in duplicates. This doesn't impact performance as we only use this model to render HTML in liquid tags, which then it's saved in the DB as processed HTML.

Reply:
Hm, since you're using `delete_all` and not `destroy_all` I don't think we need `in_batches` here. Usually, you do that to keep memory consumption from object instantiation lower, but we don't instantiate anything here anyway.

Reply:
I actually copy-pasted from https://api.rubyonrails.org/v5.2/classes/ActiveRecord/Batches.html#method-i-in_batches but I think I'll keep it as it's not about loading the objects, but to issue `DELETE`s in batches instead of one large `DELETE`